src/arrayMethodSort.js

```diff
@@ -5,7 +5,47 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    if (this.length < 2) {
+      return this;
+    }
+
+    const isFunc = compareFunction !== undefined ? 1 : 0;
```

Comment:
I don't think that you need this variable

Reply:
Yes, I agree, it can be done without this variable:)